aptos-move/move-examples/token_objects/knight/sources/food.move

```diff
@@ -0,0 +1,327 @@
+/// This module implements the knight token (non-fungible token) and the food tokens (fungible token).
```

Comment:
This comment needs to be updated for both food and knight modules

Reply:
Done. Thanks!